src/ui_bagl.c

```diff
@@ -74,10 +79,35 @@ UX_FLOW(ux_idle_flow,
         &ux_version_step,
         &ux_chain_id_step,
         &ux_authorized_key_step,
+        &ux_settings_step,
         &ux_hwm_step,
```

Comment:
I think the settings should be placed right before the exit screen.

Reply:
Sure. Makes sense.